scripts/deploy.sh

```
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+# 🛠️ 배포에 사용할 변수들
+REMOTE_APP_DIR="~/spring-roomescape-payment"
```

Shellcheck (0.10.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)